### PR TITLE
Add JS docs for mapboxgl.workerUrl

### PR DESCRIPTION
**Pull request by arindam1993**
Addresses https://github.com/mapbox/mapbox-gl-js/issues/10468

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -173,7 +173,23 @@ const exported = {
...
+     *
+     * @var {string} workerUrl
+     * @returns {string} A URL hosting a JavaScript bundle for mapbox-gl's WebWorker.
+     * @example
+     * <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp.js'></script>
+     * <script>
+     * mapboxgl.workerUrl = "https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp-worker.js";
+     * ...
+     * </script>
+     */
```
Comment (domlet) [inline src/index.js]:
```suggestion
    /**
     * Provides an interface for loading mapbox-gl's WebWorker bundle from a self-hosted URL.
     * This needs to be set only once and before any call to `new mapboxgl.Map(..)` takes place.
     * This is useful if your site needs to operate in a strict CSP (Content Security Policy) environment
     * wherein you are not allowed to load JavaScript code from a [`Blob` URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL), which is default behavior.
     *
     * See our documentation on [CSP Directives](https://docs.mapbox.com/mapbox-gl-js/api/#csp-directives) for more details.
     *
     * @var {string} workerUrl
     * @returns {string} A URL hosting a JavaScript bundle for mapbox-gl's WebWorker.
     * @example
     * <script src='https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp.js'></script>
     * <script>
     * mapboxgl.workerUrl = "https://api.mapbox.com/mapbox-gl-js/v2.3.1/mapbox-gl-csp-worker.js";
     * ...
     * </script>
     */
```

Comment (domlet) [reply src/index.js]: @arindam1993 Two small copyedits, thank you for adding this!

**Review by domlet — APPROVED** [review]
Pending @HeyStenson review, looks good to me!

**Review by HeyStenson — APPROVED** [review]
✨

**Pull request merged**